lpr.WebAPI/Controllers/OrganisationController.cs

```diff
@@ -37,6 +37,21 @@ public class OrganisationController : ControllerBase {
       return StatusCode(200, org);
     }
 
+    [HttpGet("GetPackagesPaginated/{page}/{amount}")]
```

Comment:
Can we have a default amount and page for this endpoint? 
So when you call "GetPackagesPaginated" there is always a response.